docs/products/opensearch/reference/opensearch-limitations.md

```diff
@@ -0,0 +1,257 @@
+
+| Feature | Behavior |
+|---------|----------|
+| **User management** | You manage users through Aiven API, CLI, Console, or Terraform |
+| **Access control** | You configure access using Aiven ACLs |
+| **Permission scope** | Index-level access only |
+| **User equality** | All service users have equal privileges within their ACL permissions |
+| **Dashboard tenancy** | Private dashboards per user plus global dashboards |
+| **Password changes** | Password changes you make in the dashboard reset within 24 hours |
+
+### Security management enabled
+
+| Feature | Behavior |
+|---------|----------|
+| **User management** | You manage users directly in OpenSearch using OpenSearch Security API or dashboard |
+| **Access control** | You configure access using OpenSearch Security roles and permissions |
+| **Permission scope** | Document-level access control available |
+| **Dashboard tenancy** | Full multi-tenancy support |
+| **External authentication** | SAML and OpenID Connect supported |
+| **Aiven API support** | Limited; displays state at enablement time only |
+
+:::warning
+You cannot reverse security management after you enable it. Once enabled, you manage all
+users and permissions directly in OpenSearch.
+:::
+
+:::note
+The security plugin is always present in Aiven for OpenSearch. Security management is an
+additional feature you can enable to gain full control over security configurations.
+:::
+
+## ACL limitations
+
+### Legacy services (HAProxy-based)
+
+| Limitation | Impact |
+|------------|--------|
+| **URL-based only** | Controls only HTTP methods and URL paths |
+| **Index-level only** | Cannot control dashboard access |
+| **Top-level API rules ignored** | Rules like `_bulk/admin` are not enforced |
+
+### Security plugin services
+
+| ACL type | Behavior |
+|----------|----------|
+| **Index patterns** | Converted to OpenSearch Security roles with appropriate permissions |
+| **Top-level APIs** | Ignored; OpenSearch Security enforces index-level permissions |
+| **Access levels** | Mapped to predefined action groups (`admin`, `read`, `write`) |
+
+#### ACL access levels
+
+| ACL level | Permissions granted |
+|-----------|-------------------|
+| `admin` | Full access to matching indices |
+| `read` | Read-only access to matching indices |
+| `write` | Write access to matching indices |
+| `readwrite` | Read and write access to matching indices |
+
+## Reserved users
+
+Aiven creates and manages these special users. You cannot delete or modify their
+permissions.
+
+| Username | Purpose |
+|----------|---------|
+| `avnadmin` | Default administrator user for your service |
+| `metrics_user_datadog` | Metrics collection by Datadog integration |
+| `osd_internal_user` | Internal OpenSearch Dashboards operations |
+| `replication_user` | Cross-cluster replication |
+| `os-sec-admin` | Security management access (created when you enable security management) |
+
+## Reserved roles
+
+You cannot modify the reserved roles.
```

Comment:
Not true, if you enable the security management, nothing prevents you with modifying the provider_* roles